src/test/regress/output/multi_insert_select_behavioral_analytics_create_table.source

```diff
@@ -62,3 +62,465 @@ CREATE INDEX is_index3 ON users_table(value_1);
 CREATE INDEX is_index4 ON events_table(event_type);
 CREATE INDEX is_index5 ON users_table(value_2);
 CREATE INDEX is_index6 ON events_table(value_2);
+-- Create composite type to use in subquery pushdown
+CREATE TYPE user_composite_type AS
+(
+	tenant_id BIGINT,
+	user_id BIGINT
+);
+-- do it on the workers as well
+SELECT run_command_on_workers('
+
+	CREATE TYPE user_composite_type AS
+	(
+		tenant_id BIGINT,
+		user_id BIGINT
+	);
+');
```

Comment:
Not for this, but I think we should have a function that executes stuff both on master and workers...